app/src/main/java/com/mk/tasky/agenda/data/AgendaRepositoryImpl.kt

```diff
@@ -156,4 +156,21 @@ class AgendaRepositoryImpl(
     private suspend fun deleteTaskByIdRemotely(id: String) = resultOf {
         api.deleteTask(id)
     }
+
+    override suspend fun insertEvent(event: AgendaItem.Event, isEdit: Boolean) {
```

Comment:
What's the use of `isEdit`?

Reply:
isEdit will be used later (same as insertReminder/inserTask)